internal/builders/maven/README.md

```diff
@@ -86,25 +88,51 @@ jobs:
 
 Now, when you invoke this workflow, the Maven builder will build both your artifacts and the provenance files for them.
 
-You can also release artifacts to Maven Central by adding the following step to your workflow:
+### Releasing directly to Maven Central
+
+You can also release artifacts to Maven Central with [the slsa-github-generator Maven publisher](https://github.com/slsa-framework/slsa-github-generator/blob/main/actions/maven/publish/action.yml) by adding the following step to your workflow:
 
 ```yaml
   publish:
-    needs: build
-    uses: slsa-framework/slsa-github-generator/.github/workflows/publish_maven.yml@v1.7.0
-    with:
-      provenance-download-name: "${{ needs.build.outputs.provenance-download-name }}"
-      provenance-download-sha256: "${{ needs.build.outputs.provenance-download-sha256 }}"
-      target-download-sha256: "${{ needs.build.outputs.target-download-sha256 }}"
-    secrets:
-      maven-username: ${{ secrets.OSSRH_USERNAME }}
-      maven-password: ${{ secrets.OSSRH_PASSWORD }}
-      gpg-key-pass: ${{ secrets.GPG_PASSPHRASE }}
-      gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+    - name: publish
+      id: publish
+      uses: slsa-framework/slsa-github-generator/actions/maven/publish@v1.7.0
+      with:
+        provenance-download-name: "${{ needs.build.outputs.provenance-download-name }}"
+        provenance-download-sha256: "${{ needs.build.outputs.provenance-download-sha256 }}"
+        target-download-sha256: "${{ needs.build.outputs.target-download-sha256 }}"
+        maven-username: ${{ secrets.OSSRH_USERNAME }}
+        maven-password: ${{ secrets.OSSRH_PASSWORD }}
+        gpg-key-pass: ${{ secrets.GPG_PASSPHRASE }}
+        gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
 ```
 
 Now your workflow will build your artifacts and publish them to a staging repository in Maven Central.
 
+In the above example of the publish Action, the job that invokes the Maven builder is called `build`. The publish Action uses output from that job.
+
+#### Publisher requirements
+
+Besides adding the above workflow to your CI pipeline, you also need to add the following plugin to your `pom.xml`:
```

Comment:
If the plugin is added to the pom, will it also be retrieved and be present in the final build as well? Can we use it *only* for publication?

Follow-up: there is no way to publish without a plugin?

(Sorry, this may be a dumb question, so I don't know if maven uses everything in the pom or only when necessary)

Reply:
> Follow-up: there is no way to publish without a plugin?

Yes, but the plugin is the cleanest way to hook into the Maven build cycle. This way we avoid the overhead of implementing the logic in bash. FYI: To publish to Maven, you also add plugins to handle tasks such as signing.

> Can we use it only for publication?

Good point. That should be doable. I need to double check that. Currently it runs as part of the packaging phase which will also run if you don't deploy.